### PR TITLE
suricata sanity test

### PR DESCRIPTION
**Pull request by docandrew**

### CODE REVIEW

**Review by mikevanhemert — CHANGES_REQUESTED** [review]

File: .github/workflows/package.yaml [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+        type: string
+      BB_MIN_PACKAGE:
+        required: true
+        type: string
+
 env:
-  S3_PATH: "s3://${{ secrets.AWS_BUCKET }}/${{ github.repository }}/${{ github.head_ref || github.ref_name }}"
-  BRANCH_NAME: "${{ github.head_ref || github.ref_name }}"
-  ZARF_PACKAGE: zarf-package-suricata-amd64.tar.zst
+  ZARF_S3_PATH: "s3://${{ secrets.AWS_BUCKET }}/${{ github.repository }}/${{ github.head_ref || github.ref_name }}"
```
[the remark this answers is not in the document]
Comment (Vandiver247) [reply .github/workflows/package.yaml:17]: As discussed this morning. Will update in future PR.

File: .github/workflows/package.yaml [hunk 2]
```diff
@@ -25,7 +28,8 @@ jobs:
       - name: Install Zarf
         uses: defenseunicorns/setup-zarf@main
         with:
-          version: v0.22.2
+          version: v0.23.0
```
[the remark this answers is not in the document]
Comment (Vandiver247) [reply .github/workflows/package.yaml:32]: Updated.

File: .github/workflows/package.yaml [hunk 3]
```diff
@@ -47,6 +51,39 @@ jobs:
...
+          aws s3 cp --no-progress "${{ env.BB_MIN_S3_PATH }}/${{ inputs.BB_MIN_PACKAGE }}" "${{ inputs.BB_MIN_PACKAGE }}"
+
+      - name: Install dependencies for terratest
+        run: |
+          go get -t ./...
+
+      - name: Run component test
+        working-directory: ./test
+        run: |
+          go test
```
[the remark this answers is not in the document]
Comment (Vandiver247) [reply .github/workflows/package.yaml:78]: Added BRANCH_NAME line similar to Arkime.

File: zarf.yaml [hunk 4]
```diff
@@ -5,6 +5,10 @@ metadata:
   architecture: amd64
 
 variables:
+  - name: DOMAIN
```
[the remark this answers is not in the document]
Comment (Vandiver247) [reply zarf.yaml]: Removed.

**Review by mikevanhemert — APPROVED** [review]
Lookin' good!

**Pull request merged**